packages/rrweb/src/record/mutation.ts

```diff
@@ -282,7 +282,7 @@ export default class MutationBuffer {
       // ensure shadowHost is a Node, or doc.contains will throw an error
       const notInDoc =
         !this.doc.contains(n) &&
-        (rootShadowHost === null || !this.doc.contains(rootShadowHost));
+        (!rootShadowHost || !this.doc.contains(rootShadowHost));
```

Comment:
Hmm would you mind explaining how this fixes the reported issue? 
Not sure what I'm missing here but I tried reproducing this locally, by explicitly calling `this.doc.contains(undefined)` but the call seemed to return `false` and didn't crash. 

Also, any chance that this is caused by the line above (`n`)?

Reply:
Ofc chances are that in my simple test app, `doc` is handling the contains call somewhat differently than in Sentry...

Reply:
ah, oof, I thought that `contains(undefined)` would break it. If that works, then this isn't a fix at all, then it obv. is a different issue. Then I guess we'd need to do a `rootShadowHost instanceof Node` check or something along these lines 🤔